integ-test/src/test/java/org/opensearch/sql/sql/DateTimeImplementationIT.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql;
+
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+import java.io.IOException;
+
+import static org.opensearch.sql.util.MatcherUtils.rows;
+import static org.opensearch.sql.util.MatcherUtils.schema;
+import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
+import static org.opensearch.sql.util.MatcherUtils.verifySchema;
+
+public class DateTimeImplementationIT extends SQLIntegTestCase {
```

Comment:
Are there any tests that take a datetime?

Reply:
I am unsure what you mean, can you please clarify @MaxKsyunz.

Reply:
All the tests are see are `SELECT DATETIME(<string>)....`. Since `datetime` can take string or an instance of type `datetime` I was looking for tests that verify the later.

But strings are converted to datetime type, so the tests are really `SELECT DATETIME<datetime> ...`, the version that takes `<string>` is called when it fails to cast. Is that correct?

Reply:
Yes, that is correct. Ones not correctly in the expected format will be handled as a string and treated accordingly.